src/classes/SRVC_Contact_PrimaryAffiliations.cls

```diff
@@ -167,17 +173,22 @@ public virtual with sharing class SRVC_Contact_PrimaryAffiliations {
                 }
 
                 if (newContact.get(eachModel.getContactFieldApiName()) != null) {
-                         Affiliation__c afflToCreate;
-                         try {
-                             afflToCreate = new Affiliation__c(Contact__c = newContact.ID, Account__c = (Id)newContact.get(eachModel.getContactFieldApiName()), Primary__c = true);
-                         } catch (System.TypeException typeExc) {
-                             newContact.addError(typeExc.getMessage());
-                             throw new ERR_ExceptionHandler.AffAccountRecordTypeMappingsException(Label.afflAccoutMappingError);
-                         } catch (System.StringException stringExc) {
-                             newContact.addError(stringExc.getMessage());
-                             throw new ERR_ExceptionHandler.AffAccountRecordTypeMappingsException(Label.afflAccoutMappingError);
-                         }
-                         affiliationsToCreateForEachContactList.add(afflToCreate);
+                    if (eachModel.getIsValid() == false){
+                        throw new ERR_ExceptionHandler.AffAccountRecordTypeMappingsException(Label.afflAccoutMappingError);
```

Comment:
@MatthewBlanski this error is handled by constructing an AffiliationsToProcess object with a null list of List<Affiliation__c> and Set<Id> which results in a null pointer exception.  If we initialize these collections, the null pointer exception is gone, but the Account Mapping Error never appears.  Should I replace this line with newContact.addError(Label.afflAccoutMappingError) instead?